modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,79 @@
+
+[NOTE]
+====
+Velero's file system backup feature supports both Kopia and Restic, but currently OADP supports only Restic.
+====
+
+For more inforation about using Restic with OADP, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#oadp-backing-up-applications-restic_backing-up-applications[backing up applications with Restic].
+
+For more information about OADP Data Mover, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#oadp-using-data-mover-for-csi-snapshots_backing-up-applications[Using Data Mover for CSI snapshots].
+
+* You must also use Velero's link:https://velero.io/docs/main/file-system-backup/[File System Backup] to migrate data between AWS regions or between Microsoft Azure regions.
+* Velero does not support restoring data to a cluster with a _earlier_ Kubernetes version than the source cluster.
+* It is theoretically possible to migrate workloads to a destination with a _later_ Kubernetes version than the source, but you must consider the compatibility of API groups between clusters for each custom resource. If a Kubernetes version upgrade breaks the compatibility of core or native API groups, you must first update the impacted custom resources. For more information about API group versions, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html-singlr me/backup_and_restore/index#oadp-different-kubernetes-api-versions[Working with different Kubernetes API versions on the same cluster].
+
+[id="oadp-cluster-to-cluster-uid-and-gid-ranges_{context}"]
+== UID and GID ranges
+
+When you back up data from cluster and restire it to another, consider these potential issues with UID and GID ranges and mitigations for them.
+
+Summary of issues::
+The UID and GID ranges of the namespace might change on the destination cluster. OADP does not back up and restore OpenShift UID range metadata. If the backed application requires a specific UID, ensure the range is available when restored. For more information about OpenShift's UID and GID ranges, see link:https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids[A Guide to OpenShift and UIDs].
+
+Detailed description of issues::
+When you create a namespace in {product-title} by using the shell command `oc create namespace`, {product-title} assigns the namespace a unique User ID (UID) range from its available pool of UIDs, a Supplemental Group (GID) range, and unique SELinux MCS labels. This information is stored in the `metadata.annotations` field of the cluster. This information is part of the Security Context Constraints (SCC) annotations, which comprise:
```

Comment:
We need a complete sentence for the lead in to the list. Based on https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons#introductory-text. 

```suggestion
When you create a namespace in {product-title} by using the shell command `oc create namespace`, {product-title} assigns the namespace a unique User ID (UID) range from its available pool of UIDs, a Supplemental Group (GID) range, and unique SELinux MCS labels. This information is stored in the `metadata.annotations` field of the cluster. This information is part of the Security Context Constraints (SCC) annotations, which comprise the following ___(something)__:
```